src/index.js

```diff
@@ -1 +1,31 @@
-import './index.css';
\ No newline at end of file
+import './index.css';
+import { gameId, addScores, showScoresToUI } from './modules/game.js';
+
+const name = document.querySelector('#your_name');
+const score = document.querySelector('#your_score');
+const submitBtn = document.querySelector('#submit');
+const refresh = document.querySelector('.refresh-btn');
+const gameName = { name: 'Scoccer' };
+const scores = [];
+
+submitBtn.addEventListener('click', async (e) => {
+  e.preventDefault();
+  const game = {
+    user: name.value,
+    score: score.value,
+  };
+  const id = await gameId(gameName);
+  await addScores(id, game);
+  await showScoresToUI(scores);
+  name.value = '';
+  score.value = '';
+});
+
+refresh.addEventListener('click', async () => {
+  await showScoresToUI(scores);
+});
+
+window.addEventListener('DOMContentLoaded', async (event) => {
+  event.preventDefault();
+  await showScoresToUI(scores);
+});
```

Comment:
I could only submit a score once. Every time after that, it seemed like the score I submitted was posted to the API but it is not being fetched correctly. Please fix this.

Reply:
Kindly ellaborate more, may be with a short video. i didnt get well where there error is. Thanks